lightningd/peer_control.c

```diff
@@ -622,6 +623,17 @@ static void gossipd_getpeers_complete(struct subd *gossip, const u8 *msg,
 			json_array_end(response);
 		}
 
+		
+		for (size_t i = 0; i < tal_count(ids); i++) {
+			/* If no addresses, then this node announcement hasn't been recieved yet 
+			 * So no alias information either.
+			 */
+			if (nodes[i]->addresses != NULL && pubkey_eq(&nodes[i]->nodeid, &p->id)) {
+				json_add_string(response, "alias", (char*)nodes[i]->alias);
```

Comment:
I believe the issue also wants to print the colour?

Also I think GOSSIPING peers do not get their `alias` printed?

Reply:
hmm, should be, I can add that too